testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -133,6 +133,12 @@ def test_no_frames(u):
         PCA(u, select=SELECTION).run(stop=1)
 
 
+def test_can_run_frames(u):
+    atoms = u.select_atoms(SELECTION)
+    u.transfer_to_memory()
+    PCA(u, select=SELECTION).run(frames=[0, 1])
```

Comment:
Since it's a bit of a smoke test of "will this work", and we don't assert on anything, could you add `mean=True` explicitly here? This makes sure we don't accidentally not run the new changes if something happens.
```suggestion
    PCA(u, select=SELECTION, mean=True).run(frames=[0, 1])
```